packages/nimble-components/src/rich-text-mention/base/models/mention-internals.ts

```diff
@@ -51,6 +52,11 @@ export class MentionInternals {
      */
     public readonly viewElement: string;
 
+    /**
+     * Label to use as accessible name and title of mention button
+     */
+    public readonly label: string;
```

Comment:
This label is being treated as if it is statically defined. But it can change dynamically. The lang of a page and the localized strings can be updated programmatically. Instead buttonLabel should be an observable property on mention internals updated from the buttonLabel attribute on the rich-text-mention-users element

Reply:
Updated